.github/workflows/merge.yml

```diff
@@ -593,6 +593,297 @@ jobs:
+            echo "The EVM_NETWORK variable has not been set"
+            exit 1
+          else
+            echo "ANT_PEERS has been set to $ANT_PEERS"
+            echo "EVM_NETWORK has been set to $EVM_NETWORK"
+          fi
+
+      # FIXME: do this in a generic way for localtestnets
+      - name: export default secret key
+        if: matrix.os != 'windows-latest'
+        run: echo "SECRET_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80" >> $GITHUB_ENV
+        shell: bash
+      - name: Set secret key for Windows
+        if: matrix.os == 'windows-latest'
+        run: echo "SECRET_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+        shell: pwsh
+
+      # Create test data files for pointer targets
+      - name: Create test data files
+        run: |
+          echo "test_data_A" > test_data_A.txt
+          echo "test_data_B" > test_data_B.txt
+
+      # Upload test data to get addresses for pointer targets
+      - name: Upload test data A
+        run: ./target/release/ant --log-output-dest=data-dir --local file upload --public test_data_A.txt > ./upload_output_A 2>&1
+        env:
+          ANT_LOG: "v"
+        timeout-minutes: 10
+
+      - name: Parse address A (unix)
+        if: matrix.os != 'windows-latest'
+        run: |
+          ADDRESS_A=$(rg "At address: ([0-9a-f]*)" -o -r '$1' ./upload_output_A)
+          echo "ADDRESS_A=$ADDRESS_A" >> $GITHUB_ENV
+        shell: bash
+
+      - name: Parse address A (win)
+        if: matrix.os == 'windows-latest'
+        run: |
+          $ADDRESS_A = rg "At address: ([0-9a-f]*)" -o -r '$1' ./upload_output_A
+          echo "ADDRESS_A=$ADDRESS_A" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+        shell: pwsh
+
+      - name: Upload test data B
+        run: ./target/release/ant --log-output-dest=data-dir --local file upload --public test_data_B.txt > ./upload_output_B 2>&1
+        env:
+          ANT_LOG: "v"
+        timeout-minutes: 10
+
+      - name: Parse address B (unix)
+        if: matrix.os != 'windows-latest'
+        run: |
+          ADDRESS_B=$(rg "At address: ([0-9a-f]*)" -o -r '$1' ./upload_output_B)
+          echo "ADDRESS_B=$ADDRESS_B" >> $GITHUB_ENV
+        shell: bash
+
+      - name: Parse address B (win)
+        if: matrix.os == 'windows-latest'
+        run: |
+          $ADDRESS_B = rg "At address: ([0-9a-f]*)" -o -r '$1' ./upload_output_B
+          echo "ADDRESS_B=$ADDRESS_B" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+        shell: pwsh
+
+      - name: Generate pointer signing key
+        run: ./target/release/ant --log-output-dest=data-dir --local pointer generate-key
+
+      - name: Create pointer pointing to address_A
+        run: ./target/release/ant --log-output-dest=data-dir --local pointer create test_pointer ${{ env.ADDRESS_A }} --target-data-type chunk > ./pointer_create_output 2>&1
```

Comment:
Should we test with more types than chunk? Or would that be for another PR or maybe the staging tests?